### ISSUE TITLE
Terrain height building placement

### ISSUE THREAD

**Issue by JernejHabjan** [npruehs/ue4-rts#136]
Support slopes

Following building placement should be invalid:
![image](https://user-images.githubusercontent.com/21958321/95681569-d770b600-0be0-11eb-968c-5ca4d780543a.png)

But should be valid on top of a flat hill (building should sit on top of a hill not on z=0

**Comment by npruehs** [comment]
That's correct! I believe, for A Year Of Rain we added checks for collision with the terrain (or any arbitrary collision channel) for this.

**Comment by npruehs** [comment]
I'm closing this now, for tracking purposes. Feel free to re-open in case you've got any follow-up questions!
